features/bootstrap/schema/schema.json

```diff
@@ -0,0 +1,125 @@
+{
```

Comment:
This is the JSON schema that describes the structure of the JSON that our formatter generates